heartbeat/podman-etcd

```diff
@@ -532,7 +564,19 @@ listen-peer-urls: "https://${LISTEN_PEER_URLS}:2380"
 listen-metrics-urls: "https://${LISTEN_METRICS_URLS}:9978"
 metrics: extensive
 experimental-initial-corrupt-check: true
+experimental-max-learners: 1
+etcd-socket-reuse-address: $ETCD_SOCKET_REUSE_ADDRESS
+experimental-warning-apply-duration: $(convert_duration_in_nanoseconds "$ETCD_EXPERIMENTAL_WARNING_APPLY_DURATION")
+experimental-watch-progress-notify-interval: $(convert_duration_in_nanoseconds "$ETCD_EXPERIMENTAL_WATCH_PROGRESS_NOTIFY_INTERVAL")
 EOF
+
+	{
+		echo "cipher-suites:"
+		IFS=',' read -ra cipher_array <<< "$ETCD_CIPHER_SUITES"
```

Comment:
`<<<` is bash-specific, so you should avoid it if you can. If not we'll have to rename it to .in, update the hashbang to `#!@BASH_SHELL@` and add it to "configure.ac" to auto-update.

Reply:
oh, I'll try to find an alternative